quad/src/qag_par.rs

```diff
@@ -372,13 +383,32 @@ impl QagPar {
                     new_result.1[k].3,
                 ));
             }
+            if {
+                let mut bool = true;
+                for k in 0..old_result.len() {
+                    if !((old_result[k] - new_res[k]).abs() <= 0.00001 * new_res[k].abs()
+                        && new_abserr >= 0.99 * err_sum)
```

Comment:
These `0.00001` and `0.99` are parameters. If you want to keep global, define a constant top-level.

At a later stage, we can also expose even them to the user.